common/src/IO/MapReader.h

```diff
@@ -91,10 +95,35 @@ namespace TrenchBroom {
             vm::bbox3 m_worldBounds;
             Model::ModelFactory* m_factory;
 
+        private: // data populated in response to MapParser callbacks
```

Comment:
I think the class level comment should have a paragraph or two that explains the general workflow here: 1. callbacks get called with the raw data, which we just store, 2. convert the raw data to nodes (for brushes this happens in parallel), 3. post process the nodes to resolve layers and such, 4. profit!!!1